### PR TITLE
Contributes more missing belt, back, and otherwise ERRORing sprites.

### PR DESCRIPTION
**Pull request by ArcaneMusic**
## About The Pull Request

Further contributes to #51776 so that we can end the spritepocalypse.
This one covers the following sprites:
Bikehorns (Belt, suit, and... back? Why? Why are these **BACK** SPRITES?)
 - Golden
 - airhorn
 - rubber ducks (Plus plastic variant)
Memento Mori (Last missing reported neck sprite.)
Harmonica (last missing reported face sprite)
Camera (When placed on the neck after being used, showed an error, fixed.)
Staves aplenty (back sprites)
 - Locker
 - Flight
 - Necropotence
 - Sapping
 - Wipe
Pride Hammer (Back)
Mjolnir Hammer (Back)

## Why It's Good For The Game

I had 2 hours of my life I needed to waste and this felt like a good way to kill some more ERRORs. Plus I conveniently stopped right at doing any of the guns yet again and I take that as a sign my job is done.

## Changelog
:cl:
imageadd: Belt, back, and suit sprites have been added for many clownish and magical items.
/:cl:

### CODE REVIEW

File: code/game/objects/structures/watercloset.dm [hunk 1]
```diff
@@ -239,6 +239,7 @@
 	icon = 'icons/obj/watercloset.dmi'
 	icon_state = "rubberducky"
 	inhand_icon_state = "rubberducky"
+	worn_icon_state = "duck???"
```
Comment (ShizCalev) [inline code/game/objects/structures/watercloset.dm:242]: i'd prefer not to have question marks and stuff in icon names tbh.

File: code/game/objects/structures/watercloset.dm [hunk 2]
```diff
@@ -239,6 +239,7 @@
 	icon = 'icons/obj/watercloset.dmi'
 	icon_state = "rubberducky"
 	inhand_icon_state = "rubberducky"
+	worn_icon_state = "duck"
```
Comment (ShizCalev) [inline code/game/objects/structures/watercloset.dm:242]: >icon_state is duck
>inhand_icon is duck
>worn icon isn't goose

**Comment by ShizCalev** [comment]
are the belt and belt_mirror duck icons swapped by chance? just curious since they look like they're lined up with the other respective opposite files' sprites

image

**Review by Twaticus — APPROVED** [review]

**Comment by Misha600** [comment]
also i think the protokinetic crusher is fucked

**Pull request merged**